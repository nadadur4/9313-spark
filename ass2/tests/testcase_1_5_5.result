Files df_testcase_output/part-00000-4dfc8d37-3809-4010-b291-2a3791fd85f1-c000.txt and testcase_1_5_5.txt are identical
