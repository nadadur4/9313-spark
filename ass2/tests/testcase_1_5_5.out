2003:5
govt	16
council	12
court	7
nsw	7
australian	4
2004:5
govt	16
council	14
court	7
australian	4
nsw	2
2005:5
council	23
govt	18
court	9
nsw	7
australian	4
2006:5
govt	23
council	8
court	7
australian	4
nsw	3
2007:4
govt	22
council	12
court	10
nsw	10
2008:5
court	13
council	11
govt	9
nsw	8
australian	4
2009:5
court	21
govt	9
council	8
nsw	3
australian	2
2010:5
council	13
court	5
govt	3
nsw	3
australian	2
2011:5
nsw	10
court	8
australian	5
council	3
govt	2
2012:5
council	9
court	9
australian	7
nsw	5
govt	3
2013:5
australian	10
nsw	10
council	9
court	9
govt	2
2014:5
court	15
australian	13
nsw	10
council	9
govt	2
2015:5
australian	13
nsw	7
court	6
council	3
govt	1
2016:5
nsw	12
australian	10
court	8
council	2
govt	2
2017:5
australian	11
nsw	8
court	6
council	2
govt	1
2018:3
australian	9
court	9
nsw	3
2019:4
australian	8
court	4
council	1
nsw	1
2020:5
nsw	13
australian	11
court	6
council	1
govt	1
2021:4
nsw	5
australian	3
council	1
court	1
